src/services/contracts/exchange-proxy.service.ts

```diff
@@ -0,0 +1,88 @@
+import exchangeProxyAbi from '@/lib/abi/ExchangeProxy.json';
+import ConfigService, { configService } from '@/services/config/config.service';
+import { Address, SwapToken, SwapTokenType } from '../swap/swap.service';
+import { Swap } from '@balancer-labs/sor/dist/types';
+import { TransactionResponse } from '@ethersproject/abstract-provider';
+import Web3Service, { web3Service } from '../web3/web3.service';
+
+export default class ExchangeProxyService {
+  abi: any;
```

Comment:
I believe typing it as `ContractInterface` should do the trick since this is what is needed when creating a contract with ethers.

Note - you can probably leave multicall's `abi: any` because it's a bit different since its creating a new `Interface` (and not `Contract`)